apps/gpsinfo/gps-info.js

```diff
@@ -100,12 +103,18 @@ function onGPS(fix) {
     layout.maidenhead.label = "Maidenhead: "+maidenhead;
   } else {
     layout.sat.label = fix.satellites;
-    nofix = (nofix+1) % 4;
-    layout.progress.label = ".".repeat(nofix) + " ".repeat(4-nofix);
+    layout.progress.label = "in view: " + SATinView;
   }
   layout.render();
 }
 
+function onGPSraw(nmea) {
+  if (nmea.slice(3,6) == "GSV") {
```

Comment:
On BangleJS2 there is also a message which starts with $BDGSV e.g. $BDGSV,1,1,00,0*74)
Your slice also detects this message and sets SATinView to 0.

Maybe it is better if the slice is wider:
```suggestion
  if (nmea.slice(0,7) == "$GPGSV,") {
```

Reply:
Thank you for the hint, I will create a version that adds the numbers of BDGSV and GPGSV.